client/app/account/billing/main/payments/billing-payments.controller.js

```diff
@@ -1,132 +1,93 @@
-angular.module("Billing.controllers").controller("Billing.controllers.PaymentsCtrl", function ($filter, $log, $q, $scope, $timeout, $state, $translate, constants, Alerter, BillingPayments, BillingdateRangeSelection, featureAvailability, OvhApiMe) {
+angular.module("Billing.controllers").controller("Billing.controllers.PaymentsCtrl", function ($filter, $q, $scope, $state, $translate, constants, featureAvailability, OvhApiMe) {
```

Comment:
i think it's ok to use a function for a controller right ? what are the benefits of using a class here ? (class is more verbose imho because of dependency injection)

Reply:
I'm just used to it and thought it was the status quo for every manager 😅